patches.go

```diff
@@ -34,20 +34,20 @@ func Apply(p v1beta1.Patch, cp resource.Composite, cd resource.Composed, only ..
 // ApplyToObjects works like c.Apply but accepts any kind of runtime.Object
 // It might be vulnerable to conversion panics
 // (see https://github.com/crossplane/crossplane/pull/3394 for details).
-func ApplyToObjects(p v1beta1.Patch, cp, cd runtime.Object, only ...v1beta1.PatchType) error {
+func ApplyToObjects(p v1beta1.Patch, xr, cd runtime.Object, only ...v1beta1.PatchType) error {
 	if filterPatch(p, only...) {
 		return nil
 	}
 
 	switch p.GetType() {
 	case v1beta1.PatchTypeFromCompositeFieldPath:
-		return ApplyFromFieldPathPatch(p, cp, cd)
+		return ApplyFromFieldPathPatch(p, xr, cd)
 	case v1beta1.PatchTypeToCompositeFieldPath:
-		return ApplyFromFieldPathPatch(p, cd, cp)
+		return ApplyFromFieldPathPatch(p, cd, xr)
 	case v1beta1.PatchTypeCombineFromComposite:
-		return ApplyCombineFromVariablesPatch(p, cp, cd)
+		return ApplyCombineFromVariablesPatch(p, xr, cd)
 	case v1beta1.PatchTypeCombineToComposite:
-		return ApplyCombineFromVariablesPatch(p, cd, cp)
+		return ApplyCombineFromVariablesPatch(p, cd, xr)
```

Comment:
I appreciate this 😄. There's been a good chunk of time I've lost to trying to reason out the subtle cp v cd.

Reply:
Yeah, I think the `cp` for "composite" actually predates the introduction of the term "XR". Now that XR is prevalent it definitely makes more sense as a variable name.